config/gocdb_schema.xml

```diff
@@ -380,7 +380,7 @@
 	<field>
 	    <fname>EMAIL</fname>
 	    <length>255</length>
-	    <regex>/^([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}$/</regex>
+	    <regex>/^(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6})(;(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}))?$/</regex>
```

Comment:
What is the benefit of adding the `?` before the `@` symbols?

Reply:
Well it makes the `+` operator lazy, which I can't obviously see a benefit of in this case.

Reply:
Shall we remove it?